modules/ocf_irc/templates/inspircd.conf.erb

```diff
@@ -57,27 +60,22 @@
 
 <uline server="services.irc.ocf.berkeley.edu" silent="yes">
 
-
-# Passwords needed to kill/restart the IRCd while connected to it
-<power diepass="<%= @irc_creds['die_password'] -%>"
-       restartpass="<%= @irc_creds['restart_password'] -%>"
-       pause="5">
```

Comment:
this does get used, is there no replacement?

Reply:
See https://docs.inspircd.org/3/commands/#die-ltservergt

/DIE and /RESTART no longer take in a password but instead take the server name. They still require server op status like always.

Reply:
in that case, do we need to update modules/ocf_irc/files/reload-ssl.sh?

Reply:
Looking through it, I'm not sure how that uses /DIE or /RESTART

Reply:
ah, I was confusing them